styles/BoardCard.module.scss

```diff
@@ -0,0 +1,45 @@
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 450px;
```

Comment:
Can we change the dimensions of the cards when the page becomes smaller than a certain size. My main issue is when only one column can fit, leaving a lot of white space like below. `@media` css property should be able to help you with this change.
![image](https://user-images.githubusercontent.com/35786272/222833600-3e5995a5-074f-4d69-8520-44a6ecf813db.png)